client-course-schedulizer/src/components/Searchbar/Searchbar.tsx

```diff
@@ -0,0 +1,32 @@
+import { TextField } from "@material-ui/core";
+import { Autocomplete } from "@material-ui/lab";
+import React from "react";
+import "./Searchbar.scss";
+
+const styles = {
+  searchBar: {
+    width: 300,
+  },
+};
```

Comment:
We should decide if we are going to use .scss or JSS to style our components and then make it consistent. What would you like to do?

Reply:
I definitely want to use scss, but I couldn't get it to work in this case. I don't know why